File: app/lightswitchapp/app/src/main/java/kr/lightswitch/model/response/Flag.kt
package kr.lightswitch.model.response

import kotlinx.serialization.Serializable

@Serializable
data class Flag(
    val active: Boolean,
    val description: String,
    val flagId: Int,
    val maintainerName: String,
    val tags: List<Tag>,
    val title: String
)

File: app/lightswitchios/lightswitchios/Models/Response/Flag.swift
//
//  FlagResponse.swift
//  lightswitchios
//
//  Created by 김동훈 on 5/9/24.
//

import Foundation

typealias Flags = [Flag]

struct Flag: Decodable, Hashable {
    let active: Bool
    let description: String
    let flagId: Int32
    let maintainerName: String
    let tags: Tags
    let title: String
}
